docs/docs/gatsby-cli.md

```diff
@@ -36,7 +36,7 @@ gatsby new [<site-name> [<starter-url>]]
 | Argument    | Description                                                                                                                                                                                                     |
 | ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | site-name   | Your Gatsby site name, which is also used to create a project directory.                                                                                                                                        |
-| starter-url | A Gatsby starter URL or local file path. Defaults to [gatsby-starter-default](https://github.com/gatsbyjs/gatsby-starter-default); see the [Gatsby starters](/docs/gatsby-starters/) docs for more information. |
+| starter-url | A Gatsby starter URL or local file path. Defaults to [gatsby-starter-default](https://github.com/gatsbyjs/gatsby-starter-default); see the [Gatsby starters](/docs/docs/starters.md) docs for more information. |
```

Comment:
If you go to https://gatsbyjs.org/docs/docs/starters.md you'll see that the page doesn't exist. The correct URL is `/docs/starters/`. Could you update it to the correct URL?

```suggestion
| starter-url | A Gatsby starter URL or local file path. Defaults to [gatsby-starter-default](https://github.com/gatsbyjs/gatsby-starter-default); see the [Gatsby starters](/docs/starters/) docs for more information. |
```

Reply:
#21724